frontend/cypress/integration/featureFiles/kiali_sidebar_toggle.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Sidebar toggle 
+
+User opens the Overview page and toggles the main sidebar.
+
+  Background:
+    Given user is at administrator perspective
+    And user opens the overview page
+
+  @sidebar-toggle @smoke
+  Scenario: Default visibility of sidebar
+    Then user sees the sidebar 
+
+  @sidebar-toggle @smoke
+  Scenario: Tooggle visibility of the sidebar
+    When user presses the navigation toggle button
+    And user cannot see the sidebar
+    And user presses the navigation toggle button
+    Then user sees the sidebar 
```

Comment:
Since these are two different actions I'd split this up into two different scenarios and then I think it'll read better too. Something like:
```
Given the sidebar is closed
When user presses the navigation toggle button
Then user sees the sidebar
```
and
```
Given the sidebar is open
When user presses the navigation toggle button
Then user cannot see the sidebar
```
That'll also help ensure these tests won't fail if some other test left the sidebar open or closed.

Reply:
I don't think this is a big issue. There is a setup defined on the top of the Gherkin and it basically "logs in" and loads the page all over again. I refactored the code, hope everything will be fine this time.